server/master/src/main/java/org/apache/accumulo/master/replication/DistributedWorkQueueWorkAssigner.java

```diff
@@ -67,45 +67,18 @@ protected boolean isWorkRequired(Status status) {
   protected int maxQueueSize;
   protected ZooCache zooCache;
 
-  /*
-   * Getters/setters for testing purposes
-   */
-  protected AccumuloClient getClient() {
-    return client;
-  }
-
   protected void setClient(AccumuloClient client) {
     this.client = client;
   }
 
-  protected AccumuloConfiguration getConf() {
-    return conf;
-  }
-
-  protected void setConf(AccumuloConfiguration conf) {
-    this.conf = conf;
-  }
-
-  protected DistributedWorkQueue getWorkQueue() {
-    return workQueue;
-  }
-
   protected void setWorkQueue(DistributedWorkQueue workQueue) {
     this.workQueue = workQueue;
   }
 
-  protected int getMaxQueueSize() {
-    return maxQueueSize;
-  }
-
   protected void setMaxQueueSize(int maxQueueSize) {
     this.maxQueueSize = maxQueueSize;
   }
 
-  protected ZooCache getZooCache() {
-    return zooCache;
-  }
-
   protected void setZooCache(ZooCache zooCache) {
     this.zooCache = zooCache;
```

Comment:
Seems like this should be retrieved from the client context, rather than be set explicitly separately.

Reply:
It looks like this method is used for testing.  The same with setClient().

Reply:
Opened up #876